gateway_to_backend/README.md

```diff
@@ -277,6 +277,24 @@ _Even if this version is increased, the API remains backward compatible.
 It just help backends development to identify if new features are
 present in a gateway._
 
+#### Get gateway status
+
+- **Request:**
+
+    > **topic:** gw-request/get_gw_status
+    >
+    > **content:** NA
+
+- **Response:**
+
+    > **topics:** gw-response/get_gw_status/*\<gw-id\>/\<sink-id\>*
```

Comment:
Sink-id needed?

Reply:
You are right sink-id is not used and should be removed from the readme